### PR TITLE
Bump vsc-install requirement

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Comment by stdweird** [comment]
requires https://github.com/hpcugent/vsc-install/pull/6

**Comment by stdweird** [comment]
retest this please

**Comment by stdweird** [comment]
retest this please

**Comment by stdweird** [comment]
retest this please

**Comment by boegel** [comment]
easybuild-easyblocks unit test is failing with:

```
  File "/var/lib/jenkins/.easybuild_develop/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/tools/build_log.py", line 40, in <module>
    from vsc.utils.exceptions import LoggedException
ImportError: No module named exceptions
```

Is that cause by this merge, i.e. by switching to `pkg_resources.declare_namespace` rather than using `pkgutil. extend_path`?

**Comment by boegel** [comment]
OK, seems like https://github.com/hpcugent/vsc-utils/pull/24 is needed too to fix this problem.

Which is blocked by not having vsc-base 2.4.14 on PyPI yet, I'll push it out.

File: README.md [hunk 1]
```diff
@@ -5,8 +5,9 @@
 - Python 2.6 : [![Build Status](https://jenkins1.ugent.be/job/vsc-base-python26/badge/icon)](https://jenkins1.ugent.be/job/vsc-base-python26/)
 - Python 2.7 : [![Build Status](https://jenkins1.ugent.be/job/vsc-base-python27/badge/icon)](https://jenkins1.ugent.be/job/vsc-base-python27/)
 
-Common tools used within our organization.
+# Description
```
Comment (boegel) [inline README.md]: please switch to README.rst, the PyPI page for vsc-base looks terrible now

**Comment by stdweird** [comment]
@boegel i'm trying to get it woking with testpypi first, don't push it out yet

**Comment by boegel** [comment]
@stdweird: ah... too late

I figured it was a requirement for https://github.com/hpcugent/vsc-utils/pull/24, which needs to go in because the easybuild tests are broken without that. I'm working on getting EasyBuild v2.4.0 out, so I don't want to be blocked by this...

File: README.md (hunk 1 above)
Comment (JensTimmerman) [inline README.md]: well, I think PyPI should fix their page

**Pull request merged**